<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prefrences</class>
 <widget class="QDialog" name="prefrences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="colorLabel">
     <property name="text">
      <string>Colour</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="colorSelect">
     <item>
      <property name="text">
       <string>Black</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Red</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yellow</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Purple</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pink</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="timeLabel">
     <property name="text">
      <string>Time Format</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="timeFormatSelect">
     <item>
      <property name="text">
       <string>12 Hour AM/PM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24 Hour</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QFontComboBox" name="fontComboBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="fontLabel">
     <property name="text">
      <string>Font</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="fontSizeLabel">
     <property name="text">
      <string>Font Size</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="fontSizeSpinBox">
     <property name="minimum">
      <number>12</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>prefrences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>prefrences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
